WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -104,20 +107,10 @@ def index
   end
 
   def create
-    new_competition_params = params.require(:competition).permit(:name, :competition_id_to_clone)
-    @competition = Competition.new(new_competition_params.merge(
-                                   registration_open: 1.week.from_now,
-                                   registration_close: 2.weeks.from_now))
```

Comment:
I think this was added in 75de58d46e3c474f260a4c83cdbb7afcf3beda3f because `registration_open` and `registration_close` were both `NOT NULL`.

Later on (17506dd8131cb30c6a3c9bc9925f8755cee84eab), it looks like we changed these fields to be nullable, but didn't change this code to no longer set `registration_open` and `registration_close`. I support removing these defaults now.